security-insights-schema-1.0.0.yaml

```diff
@@ -130,6 +130,35 @@ properties:
                 $id: '#/properties/contribution-policy/properties/accepts-bot-pull-requests'
                 description: 'Define if the maintainers accept pull-requests generated by bots or automated tools.'
                 type: boolean
+            bots-list:
```

Comment:
Are these "bots" or are they "automated"? There's a difference between a bot generating a PR (like via AI) and a human's crafted automation creating a PR. Also "bot" has a knee-jerk negative reaction by maintainers. I think that the terminology should be instead replaced with "automated"

Reply:
It's always hard to define what is a "bot", but I agree that it has a negative reputation, better if we change `bots` to `automated-tools`?

Reply:
I think that that would make more sense.